prow/scripts/cluster-integration/kyma-upgrade-k3d-kyma-to-kyma2.sh

```diff
@@ -5,7 +5,6 @@
 #Expected common vars:
 # - JOB_TYPE - set up by prow (presubmit, postsubmit, periodic)
```

Comment:
I think this script in general `kyma-upgrade-k3d-kyma-to-kyma2.sh` is unused now? Can you check and eventually delete it?